analysis/wwz/wwz4l.py

```diff
@@ -453,11 +459,13 @@ def process(self, events):
                 f"AbsoluteStat_uncorrelated_{year}",f"RelativeJEREC1_uncorrelated_{year}",f"RelativeJEREC2_uncorrelated_{year}",f"RelativePtEC1_uncorrelated_{year}",f"RelativePtEC2_uncorrelated_{year}",
                 f"TimePtEta_uncorrelated_{year}",f"RelativeSample_uncorrelated_{year}",f"RelativeStatEC_uncorrelated_{year}",f"RelativeStatFSR_uncorrelated_{year}",f"RelativeStatHF_uncorrelated_{year}",
                 f"JER_{year}",
+                f"MET_pfunclustered_{year}",
```

Comment:
Is this part of the do_full_jec_list? I think we should actually remove the code for this, if you're ok with it. We've moved on from it, and other code would break if we tried to run it now (e.g. in correction.py, right?). I think it is cleaner and less confusing for moving forward if we just drop it. 

If we want to go back and reference how it was in the processor, GitHub will remember that for us, it will be in the history.

Reply:
Nevermind. I've just added "documentation of corrections.py" as an issue.